ts-types/set/ensure.d.ts

```diff
@@ -0,0 +1,5 @@
+import { EnsureOptions } from '../ensure';
+
+declare function ensureSet<T>(value: any, options?: EnsureOptions): Set<T>;
+declare function ensureSet(value: any, options?: EnsureOptions): Set<any>;
```

Comment:
I think we don't need this, as currently `ensureSet` will always return input set